tests-node/unit/broccoli/translation-reducer/wrap-with-namespace-if-needed.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-env node */
+'use strict';
+
+let expect = require('chai').expect;
+
+let wrapWithNamespaceIfNeeded = require('../../../../lib/broccoli/translation-reducer/utils/wrap-with-namespace-if-needed');
+
+describe('wrapWithNamespaceIfNeeded', function() {
```

Comment:
Can we add a test for folders with spaces

Reply:
Good catch. What should be the better approach to this case?

Should we throw an exception when we find a folder with white spaces?
Should we parse "foo bar" to "foo-bar"?
Should we parse "foo bar" to "foo_bar"?
Should we do nothing, left "foo bar" as "foo bar"? (This is the current behavior)

Reply:
Lets go with underscores `foo_bar`

Reply:
Done! https://github.com/ember-intl/ember-intl/pull/945/commits/9aa7bc4a8b0e3126f0f1242055eddcd878c7d9d5